build/xtask/src/dist.rs

```diff
@@ -1026,7 +1046,7 @@ fn build(
     name: impl AsRef<Path>,
     build_config: BuildConfig,
 ) -> Result<()> {
-    println!("building path {}", build_config.crate_path.display());
+    println!("building crate {}", build_config.crate_name);
```

Comment:
I love this change because it's been bugging me for a while, which is that this on master looks like this to me:

```
building path .\app\demo-stm32f4-discovery\../../task/jefe
```

both slashes, the `..`s... now this output will look good again :)